R/drive_mv.R

```diff
@@ -87,11 +93,8 @@ drive_mv <- function(file, path = NULL, name = NULL, verbose = TRUE) {
     }
   }
 
-  request <- generate_request(
```

Comment:
Comment about this line above (I can't comment directly because it's not part of this diff):

```
current_parents <- file$files_resource[[1]][["parents"]][[1]]
```

this is not going to get the actual "current parents" in the case of multiple parents. It gets just one. We need to think about this case. If there are multiple parents ... maybe we only add the new one and we message about this? My logic is that this person / file has already crossed the Rubicon into multiple parents, so why not one more? If there is exactly one parent, which will be the case 99% of the time, we do the conventional add + remove.